### ISSUE TITLE
Add `never-multi-line` to value-list-comma-space-after

### ISSUE THREAD

**Issue by Tregonia** [stylelint/stylelint#3934]
> What is the problem you're trying to solve?

Using the `value-list-comma-space-after` rule with the `always-single-line` option selected, an error is thrown for multi-line value lists.

```
// Require a single space after the commas of value lists.
// https://stylelint.io/user-guide/rules/value-list-comma-space-after/
.value-list-comma-space-after {
  // Fails as expected.
  background-size: 0,0;
  // Fails unexpectedly on the comma after #dcffa6.
  box-shadow:
    inset 0 2px 0 #dcffa6,
    0 2px 5px #000;
}
```

Nothing was found for this issue in search of issue queue.

> What solution would you like to see?

Using the `value-list-comma-space-after` rule with the `always-single-line` option selected does not show an error for multi-line value lists.

**Comment by jeddy3** [comment]
@Tregonia Thanks for the report.

I'm unable to reproduce this, though.

Given the following config:

```json
{
  "rules": {
    "value-list-comma-newline-after": "always-multi-line",
    "value-list-comma-space-after": "always-single-line"
  }
}
```

And the following Scss: 

```scss
// Require a single space after the commas of value lists.
// https://stylelint.io/user-guide/rules/value-list-comma-space-after/
.value-list-comma-space-after {
  // Fails as expected.
  background-size: 0,0;
  // Fails unexpectedly on the comma after #dcffa6.
  box-shadow:
    inset 0 2px 0 #dcffa6,
    0 2px 5px #000;
}
```

I only get one violation:

```shell
3:21 error Expected single space after "," in a single-line list (value-list-comma-space-after)
```

I believe this is the expected behaviour.

I'll close the issue for now, but feel free to comment if I've misunderstood.
